bulbs/super_features/utils.py

```diff
@@ -0,0 +1,64 @@
+from django.apps import apps
+from django.conf import settings
+
+
+def get_data_serializer(name):
+    from bulbs.super_features.models import GUIDE_TO_HOMEPAGE, GUIDE_TO_ENTRY
+    from bulbs.super_features.data_serializers import (GuideToChildSerializer,
+                                                       GuideToParentSerializer)
+    serializer = {
+        GUIDE_TO_HOMEPAGE: GuideToParentSerializer,
+        GUIDE_TO_ENTRY: GuideToChildSerializer
+    }.get(name, None)
+    if serializer is None:
+        raise KeyError('The requested SuperFeature does not have a registered serializer')
+
+    return serializer
+
+
+def get_superfeature_model():
+    from bulbs.super_features.models import BaseSuperFeature
+
+    model_name = getattr(settings, "BULBS_SUPERFEATURE_MODEL", None)
```

Comment:
So the idea is that each property must set it's own model class here, similar to the new `settings.VIDEO_MODEL`, because it will inherit from property-specific base content?